code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine_vr.dm

```diff
@@ -62,15 +62,17 @@
 /datum/reagent/vermicetol
 	name = "Vermicetol"
 	id = "vermicetol"
-	description = "A potent chemical that treats burn damage at an exceptional rate and lasts a while."
+	description = "A potent chemical that treats physical damage at an exceptional rate."
 	taste_description = "sparkles"
+	taste_mult = 3
 	reagent_state = SOLID
 	color = "#964e06"
-	overdose = 10
+	overdose = REAGENTS_OVERDOSE * 0.5
```

Comment:
Isn't this just a 'more obscure' 15, which is what it was before this latest commit?

Reply:
I did it to match other chems in the main file.